### ISSUE TITLE
check_load does not parse load5 and load15 values on macOS

### ISSUE THREAD

**Issue by MichalMMac** [nagios-plugins/nagios-plugins#344]
I've stumbled on this when using `nagios-plugins` from MacPorts.

`check_load` parses uptime output. It expects string in following forms:

```
%*[^l]load average: %lf, %lf, %lf
%*[^l]load averages: %lf, %lf, %lf
```

However macOS `uptime` output lacks commas between load values:

```
19:01  up 8 days, 20:29, 8 users, load averages: 1.27 1.21 1.29
```

I created [patch](https://github.com/macports/macports-ports/pull/1035) for MacPorts port. However it is purely for macOS. To fix this properly in `nagios-plugins` `check_load` would probably require extra `if/else` case for macOS.

**Comment by FoUStep** [comment]
This problem still persist in HomeBrew. HomeBrew installs version 2.2.1. Is it possible to get this fix not just for MacPorts but also for HomeBrew?

(High Sierra 10.13.2) with HomeBrew updated.

**Comment by FoUStep** [comment]
No one? That's too bad... :-(

**Comment by FoUStep** [comment]
This is solved in version 2.3.2. As of today it works.

**Comment by MichalMMac** [comment]
> 	check_load: Fixed for operating systems which don't have commas in their load averages (like Mac OSX)

@FoUStep Cool. Thanks for notifing me. Closing this issue.
